Readme.md

```diff
@@ -44,6 +44,15 @@ default: false
 
 Whether to show [Brotli](https://www.wikiwand.com/en/Brotli) size or not
 
+#### showBeforeSizes
+type: `boolean`
+default: false
+
+Whether to show a comparison between the `output.file` file size as it was
+and as it is now being written. Useful to compare size of a new release to
+the previous one (though note that if you run Rollup multiple times, this
+info will be lost, except if still in your terminal history).
+
```

Comment:
Now that I think about this, this will only make sense if we compare with the last published version rather than the last time  the file was build. That way during development, I can compare with the last version otherwise it makes no sense. WDYT about this?

Reply:
Not related to this PR: I just setup GitHub actions and It seems that the tests on master are failing.  Can you have a look https://github.com/ritz078/rollup-plugin-filesize/runs/589659512?check_suite_focus=true?

Reply:
Re: the tests failing, do you get any indication of which commit started the failures?

As far as comparing with the last published version, I see a few challenges with that:

1. How is one even going to figure out what the last published version was? npm doesn't have a robust enough API to quickly query for a single file out of a package; AFAICT, one would have to look at `package.json` and download and cache a fresh package to be sure, and this seems to me pretty wasteful. For alternatives, even if everyone were using Git, one could not rely on checking commits as a means of checking the last release. One could cache the output for the terminal session in which one is using `rollup-filesize-plugin`, but what determines the duration of a session, and what about cases where a user makes multiple releases within one session (e.g., if adding a patch release and then realizing they have to add another one).
2. There are in fact some uses for not checking for the last published version but instead checking, as we are now, against the last copy of the output file. For example, during development, one might see that a particular change one just added was responsible for a size increase.

Unless you have some other ideas, my suggestion would simply be for us to change the before "was" message to clarify which file size it is being compared to, e.g., "bundle size: 100KB (compared to most recent output bundle size: 99KB)" or "compared to previous output bundle size". I don't know how one can be fully unambiguous here though, as "previous" could still be mistakenly interpreted as referring to the last release.

Reply:
Though I still need to fix tests for the `reporter` PR, I've gotten the Actions/tests working for this PR with an additional commit.

Reply:
> Though I still need to fix tests for the reporter PR, I've gotten the Actions/tests working for this PR with an additional commit.

Thank you.

> How is one even going to figure out what the last published version was? npm doesn't have a robust enough API to quickly query for a single file out of a package; AFAICT, one would have to look at package.json version (assuming it has or hasn't changed since the last release) and download and cache a fresh package to be sure, and this seems to me pretty wasteful. For alternatives, even if everyone were using Git, one could not rely on checking commits as a means of checking the last release. One could cache the output for the terminal session in which one is using rollup-filesize-plugin, but what determines the duration of a session, and what about cases where a user makes multiple releases within one session (e.g., if adding a patch release and then realizing they have to add another one).

I don't see a practical use case of comparing with the last compiled file during development. The sizes will always keep changing. The only think I will want to compare is the releases.

Yes taking version from package.json makes sense to me. I don't think anyone changes the version before publishing.

> There are in fact some uses for not checking for the last published version but instead checking, as we are now, against the last copy of the output file. For example, during development, one might see that a particular change one just added was responsible for a size increase.

In that case, the behaviour should be configurable and the default should be to compare between versions. This package is being used in multiple projects so we can't ship something that is not well thought of.